tutorials/hadoop/index.mdx

```diff
@@ -181,23 +179,23 @@ MapReduce is a programming framework that allows us to perform distributed and p
 
 MapReduce consists of two distinct tasks – Map and Reduce. As the name MapReduce suggests, reducer phase takes place after mapper phase has been completed.
 
-1. Create a directory called `input` in our home directory
+1. Create a directory called `input` in our home directory:
     ```
     mkdir ~/input
     ```
 2. Copy Hadoop's configuration files into it to use those files as our data.
     ```
     cp /usr/local/hadoop/etc/hadoop/*.xml ~/input
     ```
-3. Download Hadoop example
+3. Download Hadoop example:
     ```
     wget  https://jar-download.com/cache_jars/org.apache.hadoop/hadoop-mapreduce-examples/3.0.3/jar_files.zip
     ```
-4. Unzip the file
+4. Unzip the file:
     ```
     unzip jar_files.zip
     ```
-5. Move the `jar_files` to `/usr/local/hadoop/share/hadoop/mapreduce/hadoop/`
+5. Move the `jar_files` to `/usr/local/hadoop/share/hadoop/mapreduce/hadoop/`.
     ```
     mv hadoop-mapreduce-examples-3.0.3.jar /usr/local/hadoop/share/hadoop/mapreduce/.
```

Comment:
Logically, this step should be "mv jar_files /usr/local/hadoop/share/hadoop/mapreduce/" but I don't really know for sure

Reply:
I'll trust you on this one, cause I have no idea 😅